tests/components/sunsynkweb/conftest.py

```diff
@@ -0,0 +1,186 @@
+"""Common fixtures for the Sunsynk Inverter Web tests."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, Mock, patch
+
+from pysunsynkweb.model import Installation, Plant
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.sunsynkweb.const import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.sunsynkweb.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
+
+
+@pytest.fixture
+def sensor_keys():
```

Comment:
did you forget to check in the sensor tests?

Reply:
no, they're in the badly named `test_coordinator.py`; will rename